types/zchat-browser/zchat-browser-tests.ts

```diff
@@ -5,36 +5,80 @@ const fakeFile = new File(['foo'], 'foo.txt', {
     type: 'text/plain',
 });
 
-zChat.addTags(['tag1', 'tag2']);
-zChat.clearVisitorDefaultDepartment(fakeCallback);
+zChat.init({ account_key: 'fake_account_key' });
```

Comment:
Reordered the methods here to match the order in the definitions file for easy auditing.